.eslintrc.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": ["next/core-web-vitals", "plugin:storybook/recommended"]
+  "extends": ["next/core-web-vitals", "plugin:storybook/recommended"],
+  "rules": {
+    "react/no-unescaped-entities": ["off"]
```

Comment:
I didn't want to change this code:

https://github.com/valscion/eurovision-airtable/blob/fbf3f54db82f5bc847c10666d6406c6dcde8843a/stories/Page.tsx#L41-L44

to use `&quot;args&quot;`. That's silly.